### PR TITLE
Culls lobby music and adds two more tracks

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Culls a lot of overplayed/unfitting songs, plus a few that straight up don't work anymore either from copywrite claims or removed videos.

My criteria for good lobby music is as follows:
- music classic to ss13 like Space Asshole
- atmospheric sci-fi music
- atmospheric video game music
- preferably nothing with lyrics

I've kept a lot of... awful tracks in to compromise with just removing all of these and starting over. Some of these are massively overplayed, or fit other scenarios (like Revolution already playing with revolutions)

Adds TSCATEOTU and Forest by Jeff Imam
https://soundcloud.com/jeffimam/the-sadness-club-at-the-edge-of-the-universe
https://soundcloud.com/jeffimam/forest

# Why is this good for the game?
More cohesion

# Changelog

:cl:  
tweak: Culls the lobby music list and adds two tracks by Jeff Imam
/:cl:

### CODE REVIEW

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -27,32 +24,23 @@
...
 		"https://www.youtube.com/watch?v=MPPCNM85eRY",						// The Kinks - Super Sonic Ship
 		"https://www.youtube.com/watch?v=tRcPA7Fzebw",						// David Bowie - Starman
 		"https://www.youtube.com/watch?v=zquJ6AqvVNw",						// Dungeons of Dredmor - Diggle Hell
-		"https://www.youtube.com/watch?v=cYUGOSjRs3Q",  					// Blake Robinson - Traitor's Motif
 		"https://www.youtube.com/watch?v=uiPJQgw6M_g",						// Ribbiks - Chasing Suns
 		"https://www.youtube.com/watch?v=7F_xOzLWy5U",						// Ataraxia - Deja Vuzz
 		"https://www.youtube.com/watch?v=dxDpdfzwuD4",						// The Penis (Eek!) - Surasshu
 		"https://www.youtube.com/watch?v=VJ817kvh_DM",						// Ben Prunty - FTL - Theme Song
-		"https://www.youtube.com/watch?v=7F_xOzLWy5U",						// Ataraxia - Deja Vuzz
-		"https://www.youtube.com/watch?v=hZb_6_WfquU",            			// Steam Powered Giraffe - Fire Fire
```
Comment (Moltijoe) [inline yogstation/code/controllers/subsystem/ticker.dm:55]: this one is literally about dying to a fire on a space station??

**Comment by Yarinoi** [comment]
not kirk franklins revolution..

**Comment by PibeCantina** [comment]
You are killing leaders of the free world and fire fire

I will never forget nor forgive this

**Review by Moltijoe — APPROVED** [review]
I hate you and everything you stand for

**Pull request merged**